deploy/olm-catalog/bundle/manifests/crane-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     capabilities: Seamless Upgrades
     description: Facilitates migration of container workloads from OpenShift 3.x to OpenShift 4.x
     categories: 'Modernization & Migration, OpenShift Optional'
-    containerImage: quay.io/konveyor/mig-operator-container:latest
```

Comment:
Is this a temporary image change?

Reply:
Yes, accidentally picked it up when I amended the PR with a fix. Thanks for catching it.